.NET/Microsoft.Recognizers.Definitions/English/OptionsDefinitions.cs

```diff
@@ -0,0 +1,25 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+//     
+//     Generation parameters:
+//     - DataFilename: Patterns\English\English-Options.yaml
+//     - Language: English
+//     - ClassName: OptionsDefinitions
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Microsoft.Recognizers.Definitions.English
+{
+	using System;
+	using System.Collections.Generic;
+
+	public static class OptionsDefinitions
+	{
+		public const string LangMarker = "Eng";
+		public const string TokenizerRegex = @"[^\w\d]";
+		public const string TrueRegex = @"(true|yes|yep|y|sure|ok|\uD83D\uDC4D|\uD83D\uDC4C)";
```

Comment:
Shouldn't we enforce word boundaries here?

Reply:
Added boundaries only for words (not for emoji)